ci2/ci/ci.py

```diff
@@ -1,16 +1,107 @@
+import traceback
+import json
+import os
 import uvloop
 import asyncio
 import aiodns
+import aiohttp
 from aiohttp import web
+import jinja2
+import aiohttp_jinja2
+from gidgethub import aiohttp as gh_aiohttp
+import concurrent.futures
+import batch
+
+from log import log
+from github import *
+
+with open(os.environ.get('CI2_OAUTH_TOKEN', 'oauth-token/oauth-token'), 'r') as f:
+    oauth_token = f.read().strip()
 
 uvloop.install()
 
+watched_branches = [
+    WatchedBranch(FQBranch.from_short_str(bss))
+    for bss in json.loads(os.environ.get('WATCHED_BRANCHES'))
+]
+
 app = web.Application()
+
+app['client_session'] = aiohttp.ClientSession(
+    raise_for_status=True,
+    timeout=aiohttp.ClientTimeout(total=timeout))
+app['github_client'] = gh_aiohttp.GitHubAPI(app['client_session'], 'ci2', oauth_token=oauth_token)
+app['batch_client'] = batch.aioclient.BatchClient(app['client_session'], url=os.environ.get('BATCH_SERVER_URL'))
+
 routes = web.RouteTableDef()
 
+@routes.get('/')
+@aiohttp_jinja2.template('index.html')
```

Comment:
I don't understand why GitHub is showing this if it's already merged?

Reply:
It needs to be rebased.